lib/hooks/blueprints/actions/create.js

```diff
@@ -40,7 +40,7 @@ module.exports = function createRecord (req, res) {
 				Model.subscribe(req, newInstance);
 				Model.introduce(newInstance);
 			}
-			Model.publishCreate(newInstance.toJSON(), !req.options.mirror && req);
+			Model.publishCreate(newInstance, !req.options.mirror && req);
 		}
```

Comment:
@sgress454 Well now i remember, the reason is because how pluralize method works, it needs an instance of a model. Do you think we should remove the call to pluralize and instead use a simple    `if(!_.isArray(instance)){ [instance] }`?